docs/sources/mimir/send/otel-exponential-histograms/_index.md

```diff
@@ -38,8 +38,77 @@ Use the OpenTelemetry SDK version 1.17.0 or later.
+
+     This solution allows users to switch between the explicit bucket histogram and the exponential histogram without going to a different dashboard.
+
+   - Replace the existing explicit bucket queries with modified queries. For example, replace:
+
+     ```
+     <explicit_bucket_query>
+     ```
+
+     with
+
+     ```
+     <exponential_query> or <explicit_bucket_query>
+     ```
+
+     Where `explicit_bucket_query` is the original query and `exponential_query` is the same query using exponential histogram query syntax.
+
+     {{< admonition type="warning" >}}
+     Using the PromQL operator `or` can lead to unexpected results. For example, if a query uses a range of seven days, such as `sum(rate(http_request_duration_seconds[7d]))`, then this query returns a value as soon as there are two exponential histograms samples present before the end time specified in the query. In this case, the seven day rate is calculated from a couple of minutes, rather than seven days, worth of data. This results in an inaccuracy in the graph around the time you started scraping exponential histograms.
+     {{< /admonition >}}
+
+1. Start adding new recording rules and alerts to use exponential histograms. Don't remove the old recording rules and alerts at this time.
+1. It's important to keep scraping both explicit bucket and exponential histograms for at least the period of the longest range in your recording rules and alerts, plus one day. This is the minimum amount of time, but it's recommended to keep scraping both data types until you can verify the new rules and alerts.
+
+   For example, if you have an alert that calculates the rate of requests, such as `sum(rate(http_request_duration_seconds[7d]))`, this query looks at the data from the last seven days plus the Prometheus [lookback period](https://prometheus.io/docs/prometheus/latest/querying/basics/#staleness). When you start sending exponential histograms, the data isn't there for the entire seven days, and therefore, the results might be unreliable for alerting.
+
+1. After configuring exponential histogram collection, choose one of the following ways to stop collecting explicit bucket histograms.
+
+   - Remove the custom bucket definition, `Buckets`/`classicUpperBounds`, from the instrumentation. In Java, also use the `nativeOnly()` option. Refer to the examples in [Instrument application with Prometheus client libraries](#instrument-application-with-prometheus-client-libraries).
+   - Drop the explicit bucket histogram series with [Prometheus relabeling](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#relabel_config) or [Grafana Alloy prometheus.relabel](https://grafana.com/docs/alloy/<ALLOY_VERSION>/reference/components/prometheus/prometheus.relabel) at the time of scraping.
```

Comment:
I haven't tried it but in the Otel collector you have something like [filter processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/filterprocessor/README.md) that should be possible to utilize for the same purpose. Let's not mix Prometheus or Alloy here for now.